Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM mozillamarketplace/centos-mysql-mkt:0.2
 
+# Copy the most recent git HEAD information so that we can debug
+# the state of the docker image better
+COPY .git/logs/HEAD /git-rev.txt
```

Comment:
Also, because this command will change pretty every time a new image is built it should go towards the end of the file. That way it won't invalidate all the other layers. This would help keep things fast when pulling new image layers.